source/catalog.html

```diff
@@ -0,0 +1,257 @@
+              Cat Energy SLIM 500 г
+            </h4>
+            <dl class="card__description description-table">
+              <dt class="description-table__element">Масса</dt>
+              <dd class="description-table__value">500 г</dd>
+              <dt class="description-table__element">Вкус</dt>
+              <dd class="description-table__value">Рис</dd>
+              <dt class="description-table__element">Цена</dt>
+              <dd class="description-table__value">500 ₽</dd>
+            </dl>
+            <button class="card__button button" type="button">Заказать</button>
+          </div>
+        </div>
+      </section>
+
+      <section class="catalog-container__section show-more-section">
+        <h4 class="show-more-section__title">Показать ещё 100500 товаров</h4>
+        <p class="show-more-section__description">На самом деле вкусов гораздо больше!</p>
+        <button class="show-more-section__button button" type="button">Показать все</button>
+      </section>
+
+      <section class="catalog-container__section additional-products">
+        <div class="additional-products__card additional-card">
+          <h4 class="additional-card__title">Сахарозаменитель</h4>
+          <dl class="additional-card__table description-table">
+            <dt class="description-table__element">1 упаковка (100 г)</dt>
+            <dd class="description-table__value">200 ₽</dd>
+          </dl>
+        <button class="additional-card-button button">Заказать</button>
+        </div>
+
+        <div class="additional-products__card additional-card">
+          <h4 class="additional-card__title">Питьевая вода</h4>
+          <dl class="additional-card__table description-table">
+            <dt class="description-table__element">5 литров</dt>
+            <dd class="description-table__value">50 ₽</dd>
+          </dl>
+        <button class="additional-card-button button">Заказать</button>
+        </div>
+
+        <div class="additional-products__card additional-card">
+          <h4 class="additional-card__title">Молоко</h4>
+          <dl class="additional-card__table description-table">
+            <dt class="description-table__element">1 литр</dt>
+            <dd class="description-table__value">100 ₽</dd>
+          </dl>
+        <button class="additional-card-button button">Заказать</button>
+        </div>
+
+        <div class="additional-products__card additional-card">
+          <h4 class="additional-card__title">Витамины</h4>
+          <dl class="additional-card__table description-table">
+            <dt class="description-table__element">1 упаковка (30 г)</dt>
+            <dd class="description-table__value">300 ₽</dd>
+          </dl>
+        <button class="additional-card-button button">Заказать</button>
+        </div>
+      </section>
+
+      <section class="catalog-container__section gift-section">
+          <p class="gift-section__text">Закажите все и получите чехол для кота в подарок!</p>
```

Comment:
Почему отступ такой большой
```suggestion
        <p class="gift-section__text">Закажите все и получите чехол для кота в подарок!</p>
```

Reply:
странно, у меня в редакторе нормальный 🤔

Reply:
Что-то с ними вообще все странно, проверь установлен ли плагин editorconfig и все ли нормально там с ним работает.
Нет ли там табов местами.

Reply:
Всё установлено. Отформатировала все три файла на всякий.